src/utils/validation.ts

```diff
@@ -31,6 +31,10 @@ export const validation = {
     return guidRegEx.test(guid);
   },
 
+  isValidUserPrincipalNameArray(ups: string[]): boolean {
```

Comment:
`upns`

Reply:
ouch, UPS is definitely something different indeed 😄